virtual-setup/rstudio-login.md

```diff
@@ -1 +1,56 @@
-## RStudio Server instructions
\ No newline at end of file
+## RStudio Server Set Up
+
+We will send you your Username and temporary password by a direct message on Slack, after you are logged in on [Cancer Data Science](http://ccdatalab.org/slack).
+
+Go to https://rstudio.ccdatalab.org (You may want to bookmark this) and type in your username and temporary password, and click `Sign in`.
+
+![RStudio Login](screenshots/rstudio-server-login.png)
+
+Signing in should bring you to the RStudio session page.
+Click on the `Terminal` tab.
+
+![RStudio Session](screenshots/rstudio-session-terminal.png)
+
+Type in the `passwd` command in the `Terminal` tab.
+
+![RStudio change password](screenshots/rstudio-password-change-1.png)
+
+...and press `Enter`. Then type in the password you were given in Slack.
+
+![RStudio change password](screenshots/rstudio-password-change-2.png)
+
+Type in the new password you've chosen once, and press `Enter`, then it will ask you to confirm by typing it in again.
+Also press `Enter`.
+If you forget your password at anytime, Slack on of the CCDL team members to assist you.
+
+## Navigating the RStudio Server
+
+In the upper right corner of the session page, you should see these buttons:
+
+![RStudio Navigation](screenshots/rstudio-session-buttons.png)
+
+You can restart your R session with the orange, circular, on/off button.
+You will want to do this each time you switch to notebooks, more on this in the
+[RStudio guide instructions](../intro-to-R-tidyverse/00a-rstudio_guide.md).
+
+### About RStudio Server sessions
+
+Clicking on the house button will bring you to the RStudio server workspaces page.
```

Comment:
Note when you hit the on/off button you see this and then have to start a new session. You may want a screenshot in the above section that illustrates 1) what it looks like when you have no sessions because you ended a session 2) how you would go about starting a new session. 

Side note for possible future modifications: I think we might want to use `.Rproj` for each module so people can open a project, but we're not there yet.